server.js

```diff
@@ -38,28 +36,117 @@ var renderHtml = function(url, cb) {
             }, 10000); //If no callPhantom script was called on the page, then wait 10secs and load page anyway.
         });
     };
+
+    console.debug("Opening page");
     page.open(url);
 
     page.onError = function (msg, trace) {
-        console.log(msg);
+        console.error(msg);
         trace.forEach(function(item) {
-            console.log('  ', item.file, ':', item.line);
+            console.error('  ', item.file, ':', item.line);
```

Comment:
P.S. A simple way to make it behave differently is by including this:

```javascript
window['console']['debug'] = function(msg) {
    console.log("DEBUG: " + msg);
}
```

although that would remove option to send multiple arguments, but you catch my drift.

To disable debug logging, you could do

```javascript
window['console']['debug'] = function() {};
```